### PR TITLE
Bob/4002 touchpoints in landmark

### LINKED ISSUE #4002: Touchpoints modal is not contained by a landmark in the app
## Area or Component
Application-wide

## Quick Summary
The touchpoints modal is not part of the page's landmarks (i.e., `<header>, <footer>, or <main>`.) 

### How to Replicate
Inspect touchpoints modal in-app

## Accessibility Violation
All page content should be contained by landmarks: https://dequeuniversity.com/rules/axe/4.4/region?application=AxeChrome

## Action Requested
Add Touchpoints to the page landmarks. This might require us to create an application-wide footer.

## Original Deque Issue
  | region | Ensures all page content is contained by landmarks | Touchpoints modal not contained by landmark | All page content should be contained by landmarks | https://dequeuniversity.com/rules/axe/4.4/region?application=AxeChrome | moderate | No | No |   | [".touchpoints-wrapper"] | Fix any of the following:  Some page content is not contained by landmarks | <div class="touchpoints-wrapper sr-app-touchpoints-button">How can we improve SimpleReport?</div> | cat.keyboard,best-practice | rnv7@cdc.gov | Sign up selector | https://www.simplereport.gov/app/sign-up/ |   | 2022-07-15T19:09:47.072Z |   |   |   |   |   |   |   |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

### PR DESCRIPTION
**Pull request by fzhao99**
# FRONTEND PULL REQUEST

## Related Issue

Fixes #4002 

## Changes Proposed

- Adds a footer in the page component app-wide
- Sticks touchpoints button and modal into that component
 
## Testing

- Verify touchpoints works as expected
- Verify the modal is in the landmark

Implementation is live in [pentest](https://pentest.simplereport.gov/app/) for testing

## Screenshots / Demos

Screen Shot 2022-09-06 at 9 58 31 AM

## Checklist for Author and Reviewer
- [ ] Verify touchpoints works as expected
- [ ] Verify the modal is in the landmark

### Documentation
- [ ] Any changes to the startup configuration have been documented in the README

### CODE REVIEW

File: frontend/public/touchpoints.js [hunk 1]
```diff
@@ -822,4 +821,4 @@ const touchpointsFormHtmlString = `<div id="fba-modal-dialog" class="fba-modal-d
 </div>
 `;
 
-const touchpointForm5b1efe87 = new FBAform(document, window).init(formOptions);
+new FBAform(document, window).init(formOptions);
```
Comment (emmastephenson) [inline frontend/public/touchpoints.js:824]: Looks like LGTM is annoyed about this `window` param - I think it can be safely removed.

**Review by emmastephenson — DISMISSED** [review]
Overall LGTM! Just one comment about the `window` param.

**Review by georgehager — APPROVED** [review]
Checked it out in pentest, looks good!

**Review by georgehager — DISMISSED** [review]
Thanks for the update for LGTM!

**Review by emmastephenson — DISMISSED** [review]
LGTM, thanks Bob!

**Review by BobanL — APPROVED** [review]
Looks great! Thanks for adding the isModalActive on the footer 😃

**Review by emyl3 — APPROVED** [review]
LGTM! Thank you bob!

**Pull request merged**